napari/utils/io.py

```diff
@@ -16,8 +17,27 @@ def imsave(filename: str, data: np.ndarray):
     data : np.ndarray
         The image data.
     """
-    ext = os.path.splitext(filename)[1]
-    if ext in [".tif", ".tiff"]:
+    ext = os.path.splitext(filename)[1].lower()
+    # If no file extension was specified, choose .png by default
+    if ext == "":
+        ext = ".png"
+    # Save screenshot image data to output file
+    if ext in [".png"]:
+        import PIL
+        from imageio.v3 import imwrite
+
+        pnginfo = PIL.PngImagePlugin.PngInfo()
+        pnginfo.add_text(
+            "Software", f"napari version {__version__} https://napari.org/"
+        )
+        imwrite(
+            filename,
+            data,
+            extension='.png',
+            plugin='pillow',
+            pnginfo=pnginfo,
+        )
```

Comment:
Would it be useful to have this as a separate `imwrite_png`, that can be use both here and in `notebook_display.py` to avoid potential duplication ?

Reply:
Done